general/development/gettingstarted.md

```diff
@@ -10,8 +10,8 @@ tags:
 - [Set up your development environment](#a-quick-start-to-moodle-development)
 - See the [list of relatively easy Moodle bugs](https://tracker.moodle.org/issues/?jql=project%20%3D%20MDL%20AND%20type%20in%20(bug)%20AND%20status%20%3D%20open%20AND%20Difficulty%20%3D%20Easy%20AND%20labels%20not%20in%20(patch)%20ORDER%20BY%20created%20DESC)
 - [Prepare a patch](https://docs.moodle.org/dev/Git_for_developers#Preparing_a_patch)
-- Read the [Coding style](/general/development/policies/codingstyle) policy
-- Send your patch to [peer review](/general/development/process/peer-review) and learn more about the [Moodle processes](/general/development/process).
+- Read the [Coding style](./policies/codingstyle/index.md) policy
+- Send your patch to [peer review](/general/development/process/peer-review) and learn more about the [Moodle processes](./process.md).
```

Comment:
```suggestion
- Send your patch to [peer review](./process/peer-review.md) and learn more about the [Moodle processes](./process.md).
```

Reply:
Thanks - there was a bug in the regular expression which meant that only the last link on a line was captured.